packages/wrangler/src/deploy/index.ts

```diff
@@ -300,6 +336,10 @@ export async function deployHandler(
 					args.siteExclude
 				);
 
+	//TODO: decide what to do with multiple directories
```

Comment:
If these multiple directories are separate bindings, why are we passing them through as one directory? Better the single directory from args gets added as an entry of this array (with a default binding name eg ASSETS?) and this whole array is passed through

Reply:
ah thx for catching this!! I meant to add a comment as well when I reviewed Emily's changes!

TBH I don't think this is something we want to support in the first iteration...not as I understood it at least. For now we want to support the simplest and most straightforward use case, aka 
`wrangler de --experimental-assets <directory>` and `wrangler deploy --experimental-assets <directory>`.

I'll remove this comment!